wrf-cmake.rb

```diff
@@ -0,0 +1,73 @@
+# WRF-CMake (https://github.com/WRF-CMake/WRF).
+# Copyright 2019 M. Riechert and D. Meyer. Licensed under the MIT License.
+
+class WrfCmake < Formula
+  desc "The Weather Research and Forecasting (WRF) model with CMake support"
+  homepage "https://github.com/WRF-CMake/WRF"
+  url "https://github.com/WRF-CMake/WRF.git",
+      :branch => "wrf-cmake",
+      :commit => "8b9edc944f0cbbfb22966850ae3724f617f0f705"
+  version "4.1.wrf-cmake"
+  sha256 "37814ee7bfe7077cba8bd0175258ef763380fce3e7d8aab2ea8360a902a11043"
+  head "https://github.com/WRF-CMake/WRF.git", :branch => "wrf-cmake"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" # for gfortran
+  depends_on "jasper"
+  depends_on "libpng"
+  depends_on "netcdf"
+  depends_on "open-mpi"
+
+  resource "WPS" do
+    url "https://github.com/WRF-CMake/WPS.git",
+        :branch => "wps-cmake",
+        :commit => "d918dc6da17b177aa1fe0082866274968f45a171"
+  end
+
+  # Note: If you get an internal compiler error when building WRF,
+  # then you're likely running out of memory. To reduce memory usage,
+  # set HOMEBREW_MAKE_JOBS=1.
+
+  def install
+    (buildpath/"WPS").install resource("WPS")
+
+    # Prevent CMake from finding & using ifort
+    ENV["FC"] = Formula["gcc"].opt_bin/"gfortran"
+
+    wrf_args = std_cmake_args.map do |s|
+      s.sub /-DCMAKE_INSTALL_PREFIX.*$/, "-DCMAKE_INSTALL_PREFIX=#{prefix}/wrf"
```

Comment:
Why not use the approach I used below for WPS? Overriding works just fine and looks far simpler.

Reply:
Yes, that works. I didn't realize you were also passing `std_cmake_args` for WPS. I'm happy to do that, but it does rely on the behavior of CMake that CMake definitions on the right of the command line override those on the left. (`-DCMAKE_INSTALL_PREFIX` is being passed twice to CMake) I'm more confident of the stability of Ruby syntax than I am of the semantics of CMake command line defines. At the end of the day both implementations are effectively the same, but I agree that they should probably match. I'll go ahead and change this to match WPS, since it sounds like you prefer that.